src/main/types/CourtAddress.ts

```diff
@@ -0,0 +1,35 @@
+
+export interface DisplayCourtAddresses {
+  primary: DisplayAddress;
+  secondary: DisplayAddress
+}
+
+export interface DisplayAddress {
```

Comment:
Can you reuse CourtAddress for this? They seem identical except for the address_lines and address_lines_cy parts :)

Reply:
The difference in the address 'lines' is the reason they're separate types. The API expects and returns address lines in an array. The front-end requirements/screenshot/GDS component expect a single string.